docs/basic_training/intro.es.md

```diff
@@ -0,0 +1,274 @@
+
+</figure>
+
+### Lenguaje de escritura
+
+Nextflow implementa un DSL declarativo que simplifica la escritura de flujos de trabajo de análisis de datos complejos como una extensión de un lenguaje de programación de propósito general.
+
+Este enfoque hace que Nextflow sea flexible: brinda los beneficios de un DSL conciso para el manejo de casos de uso frecuente con facilidad **y** da la flexibilidad y el poder de un lenguaje de programación de propósito general para manejar casos más complejos en el mismo entorno. Esto sería difícil de implementar utilizando un enfoque puramente declarativo.
+
+En términos prácticos, las secuencias de comandos de Nextflow son una extensión del [lenguaje de programación Groovy] (https://groovy-lang.org/) que, a su vez, es un superconjunto del lenguaje de programación Java. Groovy puede considerarse como "Python para Java", ya que simplifica la escritura de código y es más accesible.
+
+## Tu primer script
+
+Aquí ejecutará su primer script de Nextflow (`hello.nf`), que revisaremos línea por línea.
+
+En este ejemplo sencillo, el script toma una cadena de entrada (un parámetro llamado `params.saludo`) y la divide en partes de seis caracteres en el primer proceso. El segundo proceso convierte los caracteres a mayúsculas. El resultado finalmente se muestra en la pantalla.
+
+### Nextflow code
+
+<!-- NOTE: (Phil, Jan 2023)
+We can dynamically include external files using mkdocs, as follows:
+
+```groovy title="nf-training/hello.nf" linenums="1"
+--8<-- "nf-training/hello.nf"
+```
+
+This inserts a code snippet identical to the one below, and we don't have to worry about keeping the two in sync.
+
+HOWEVER - currently the line annotations cannot be added for external files. So for now, we still need to copy the scripts.
+
+TODO: Maybe either:
+    - Rewrite docs to not use loads of annotations
+    - Wait for future versions to allow annotations with external files
+-->
+
+!!! info
+
+    Click the :material-plus-circle: icons in the code for explanations.
+
+```groovy title="nf-training/hello.nf" linenums="1"
+#!/usr/bin/env nextflow
+// (1)!
+
+params.saludo = '¡Hola mundo!' // (2)!
+saludo_ch = Channel.of(params.saludo) // (3)!
+
+process SEPARALETRAS { // (4)!
+    input: // (5)!
+    val x // (6)!
+
+    output: // (7)!
+    path 'segmento_*' // (8)!
+
+    // (9)!
+    """
+    printf '$x' | split -b 6 - segmento_
+    """
+} // (10)!
+
+process MAYUSCULAS { // (11)!
+    input: // (12)!
+    path y // (13)!
+
+    output: // (14)!
+    stdout // (15)!
+
+    // (16)!
+    """
+    cat $y | tr '[a-z]' '[A-Z]'
+    """
+} // (17)!
+
+workflow { // (18)!
+    letras_ch = SEPARALETRAS(saludo_ch) // (19)!
+    resultados = MAYUSCULAS(letras_ch.flatten()) // (20)!
+    resultados.view{ it } // (21)!
+} // (22)!
+```
+
+1. El código comienza con un "shebang", que declara a Nextflow como intérprete.
+2. Declara un parámetro 'saludo' que se inicializa con el valor 'Hello world!'.
```

Comment:
If we leave the spanish translation for the process definition, then **params.saludo is '¡Hola mundo!'**